mdesnoyer/wonderland
# Kill Bulma

# Changes
- updated `package.json`
- remove Bulma dependent CSS
# Test Plan
- make sure site builds
- make sure site looks OK
